test/parallel/xyk-pallet.API.fees.tripleTokens.test.ts

```diff
@@ -66,32 +72,25 @@ beforeAll(async () => {
 
   keyring.addPair(sudo.keyRingPair);
 
+  await updateFeeLockMetadata(sudo, null, null, null, [
+    [firstCurrency, true],
+    [secondCurrency, true],
+  ]);
+
```

Comment:
perhaps this conflicts with some other setups, are this tests  running in parallel'?

Reply:
Tests were checked in parallel, all worked well.